homeassistant/components/duckdns/__init__.py

```diff
@@ -111,11 +121,9 @@ async def _update_duckdns(session, domain, token, *, txt=_SENTINEL, clear=False)
 def async_track_time_interval_backoff(
     hass: HomeAssistant,
     action: Callable[[datetime], Coroutine[Any, Any, bool]],
-    intervals,
+    intervals: Sequence[timedelta],
 ) -> CALLBACK_TYPE:
     """Add a listener that fires repetitively at every timedelta interval."""
-    if not isinstance(intervals, (list, tuple)):
-        intervals = (intervals,)
```

Comment:
The method is only ever called with a tuple of values for `intervals`. So it's safe to remove this check here and simplify the typing.

https://github.com/home-assistant/core/blob/7a6dca098782c03b829695721ed49b3b1fac39e5/homeassistant/components/duckdns/__init__.py#L61-L68